apps/yarn.lock

```diff
@@ -60,7 +60,7 @@
     temporal latest
   optionalDependencies:
     browser-serialport latest
-    firmata "^0.12.0"
+    firmata latest
```

Comment:
It unfortunately is the case that `0.16.0` is not included in the range `^0.11.3`, and the upstream `playground-io` has itself not been updated from the latter. Can you check whether updating it to `0.16.0` breaks anything?